packages/api-explorer/specs/Looker.3.1.oas.json

```diff
@@ -2,18 +2,15 @@
   "openapi": "3.0.0",
   "info": {
     "version": "3.1.0",
-    "x-looker-release-version": "7.13.0",
+    "x-looker-release-version": "7.15.0",
```

Comment:
Sorry, I'm gonna assume that all the changes to this particular file are valid. Might want to get another approval on this one.

Reply:
yeah, I refreshed the spec files. These won't be in the final version at all because specs will be fetched directly from the Looker server. We've got some Looker server changes to land before that's possible, though.